scene/gui/graph_edit.cpp

```diff
@@ -1453,6 +1459,13 @@ void GraphEdit::force_connection_drag_end() {
 	emit_signal(SNAME("connection_drag_ended"));
 }
 
+bool GraphEdit::is_node_hover_valid(const StringName &p_from, const int p_from_port, const StringName &p_to, const int p_to_port) {
+	if (get_script_instance() && get_script_instance()->has_method(SNAME("_is_node_hover_valid"))) {
+		return get_script_instance()->call(SNAME("_is_node_hover_valid"), p_from, p_from_port, p_to, p_to_port);
+	}
+	return true;
```

Comment:
You should use `GDVIRTUAL_CALL` instead:
```suggestion
	bool valid;
	if (GDVIRTUAL_CALL(_is_node_hover_valid, p_from, p_from_port, p_to, p_to_port, valid)) {
		return valid;
	}
	return true;
```